backend/app/modules/uq/service/preliminary.py

```diff
@@ -52,12 +51,15 @@ def retrieve(
 
     def calculate(self, context: Context, user_address: str) -> Decimal:
         gp_score = self._get_gp_score(context, user_address)
-        budget = self.budgets.get_budget(context, user_address)
-        addresses = (
-            app.config["ADDRESSES"].split(",") if app.config["ADDRESSES"] else []
-        )
 
-        return calculate_uq(gp_score, budget, user_address, addresses)
+        return calculate_uq(gp_score)
+
+        # Previous logic by cheking the budget and addresses blacklist
+        # budget = self.budgets.get_budget(context, user_address)
+        # addresses = (
+        #     app.config["ADDRESSES"].split(",") if app.config["ADDRESSES"] else []
```

Comment:
since we don't use ADDRESSES env enymore you can also remove it from such files as settings.py etc.